tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -46,6 +46,9 @@ def delete_and_create_osd_node(osd_node_name):
 
```

Comment:
does this test run with IOs in background during node replacement? If not, please add IOs

Reply:
We have 2 tests in this file: 
1. https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py#L100
2. https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py#L165

The first one is with IO in the background and the second one without IO in the background. 
Currently, with AWS UPI and vSphere, I ran it without IO cause the rebalancing took too long. But I added the param `--io-in-bg --io-load 30` in the test.

Reply:
will be done as part of https://projects.engineering.redhat.com/browse/OCSQE-578